lib/validation/JsonSchemaValidator.js

```diff
@@ -9,9 +8,7 @@ class JsonSchemaValidator {
     this.ajv = ajv;
 
     // TODO Validator shouldn't know about schemas
-
-    this.ajv.addMetaSchema(dashSchema);
-
+    // this.ajv.addMetaSchema(require('ajv/lib/refs/json-schema-draft-07'));
```

Comment:
@shumkov Just checking if this comment here is for a reason and not a typo or something.